extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -3429,4 +3417,18 @@ static public function getVariableForAllWikis( $variableName, $limit, $afterWiki
 		$db->freeResult( $dbResult );
 		return $result;
 	}
+        
+        /**
+         * unserializeErrorHandler
+         *
+         * @author Emil Podlaszewski <emil@wikia-inc.com>
+         */
+        static public function unserializeHandler( $errno, $errstr ) {
```

Comment:
At this point the `WikiaLogger` should be set up already, we could just rely on that instead of this custom logger.

Reply:
Addressed here: https://github.com/Wikia/app/pull/15106/commits/9fd38203584eeb976017950b5363f8a38f5bfd62